lib/rules/require-parentheses-around-arrow-param.js

```diff
@@ -48,6 +48,9 @@ module.exports.prototype = {
     check: function(file, errors) {
         function isWrapped(node) {
             var openParensToken = file.getPrevToken(file.getFirstNodeToken(node));
+            if (openParensToken.value === '...') {
```

Comment:
I think it's because there is 2 parens `((` so it would have the same result if you prevToken.